tests/integration/components/operator-form/include-operator-select/component-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('operator-form/include-operator-select', 'Integration | Component | operator form/include operator select', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
+
+  this.render(hbs`{{operator-form/include-operator-select}}`);
+
+  assert.equal(this.$().text().trim(), '');
+
+  // Template block usage:" + EOL +
+  this.render(hbs`
+    {{#operator-form/include-operator-select}}
```

Comment:
Expected ')' and instead saw '{'.<br>Missing semicolon.<br>Expected an identifier and instead saw '#'.<br>Expected an assignment or function call and instead saw an expression.<br>'operator' is not defined.<br>'form' is not defined.<br>'include' is not defined.<br>'select' is not defined.